app/models/course/condition.rb

```diff
@@ -5,4 +5,9 @@ class Course::Condition < ActiveRecord::Base
   belongs_to :conditional, polymorphic: true
 
   delegate :satisfied_by?, to: :actable
+
+  ALL_CONDITIONS = [
+    Course::Condition::Achievement.class_name.downcase,
```

Comment:
the condition's name needs to be downcase because `url_for` needs downcased strings.